### PR TITLE
docs(mesh/rbac): add information how to unlock yourself

### PR DESCRIPTION
**Pull request by lukidzi**
### Description

Added docs how to unlock yourself in case you remove default rbac

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

**Review by lobkovilya — APPROVED** [review]

File: app/_src/mesh/features/rbac.md
Comment (jakubdyszkiewicz) [inline app/_src/mesh/features/rbac.md:1192]: Please use `configure the control plane` and put a link to https://docs.konghq.com/mesh/latest/documentation/configuration/#modifying-the-configuration

**Review by jakubdyszkiewicz — APPROVED** [review]

File: app/_src/mesh/features/rbac.md
Comment (cloudjumpercat) [inline app/_src/mesh/features/rbac.md:1190]:
```suggestion
If you remove the default `AccessRoleBinding` and `AccessRole`, you might find yourself locked out and unable to edit any resources. If you encounter this situation, you can regain access to the cluster by following these steps:
```

File: app/_src/mesh/features/rbac.md
Comment (cloudjumpercat) [inline app/_src/mesh/features/rbac.md:1187]:
```suggestion

{% if_version gte:2.0.x %}
```

File: app/_src/mesh/features/rbac.md
Comment (cloudjumpercat) [inline app/_src/mesh/features/rbac.md:1194]:
```suggestion
3. Remove the `KUMA_ACCESS_TYPE` environment variable for the control-plane and restart the control-plane.
{% endif_version %}
```

**Comment by cloudjumpercat** [comment]
@lukidzi Hi! The broken links test should be fixed once https://github.com/Kong/kong-mesh/pull/5428 is merged. And the failing smoke tests are expected since that happens on PRs from a fork. I'll get this approved once the broken links test passes after that PR is merged.

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for making this update! The one smoke test is expected to fail because this PR is from a fork.

**Pull request merged**
